src/Translator/src/DependencyInjection/TranslatorCompilerPass.php

```diff
@@ -0,0 +1,32 @@
+<?php
+
+declare(strict_types=1);
```

Comment:
```suggestion

```

There is no strict types in the Symfony codebase afaik

Reply:
Alright, thanks! Removed the strict types.